src/agent/script.yaml

```diff
@@ -263,6 +264,37 @@
         variable: event_description
     - say: המידע החדש נשמר
 
+  - name: 'מאבטח/ת: בדיקה אילו פרטים חסרים צריך להשלים'
+    steps:
+    - do:
+        cmd: getGuardCompany
```

Comment:
@noamoss 
It would be simpler here to change the method to return if `offender_organization` has a value (like so:
```
return !!record.offender_organization;
```

Then switch on `match: false` instead.